grails-app/services/au/org/ala/spatial/service/ServiceAuthService.groovy

```diff
@@ -87,5 +94,59 @@ class ServiceAuthService {
         }
         return false
     }
+    /**
+     * Return true, if a use login (local) or a valid apikey + userId in header
+     * @return
+     */
+    boolean isAuthenticated() {
+        if (isLoggedIn()){
+            return true
+        }
+        if ( getUserId() && hasValidApiKey() ){
```

Comment:
Some concern that this might provide admin access via a proxy service that appends the apiKey.